tests/test_cli.py

```diff
@@ -1,6 +1,8 @@
 import os
 import tempfile
 
+import pybibtex
```

Comment:
What for?

Reply:
Initially put the test there, but then moved it to `test_tools`. Leftover import that I forgot to remove. Fixed now